### PR TITLE
first stab at ADR for opensearch

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:
- first stab at ADR for opensearch

## security considerations
None

### CODE REVIEW

**Comment by amhuber** [comment]
Massive thanks for continuing to look into replacing the old community logsearch release.  Everything in your proposal looks great to me.  Answers to your questions regarding how we're using the existing product:

- We're definitely hoping for something directly consumable by the community that works with the existing Prometheus release (or at least if changes are required that they are coordinated with that release)
- We definitely care about both platform and tenant logs
- The future improvements sound great but definitely aren't needed for an MVP

File: adr/ADR-0007-opensearch-replacement-for-elasticsearch.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+We do _not_ want to provide a direct migration path between these. We intend to migrate by importing logs from S3 into a new
+cluster, maintaining both during a user transition period, then sunsetting the original.
+
+We intend to offer some of the other features of ODfE in the future, such as alerting and anomaly detection, but these are not
+part of the initial launch. 
+
+## Open Questions
+
+- given that we build our bosh releases all funny, how do we make this community-friendly within our approved processes?
```
Comment (mheadd) [inline adr/ADR-0007-opensearch-replacement-for-elasticsearch.md:39]:
```suggestion
- given that we build our bosh releases all fully, how do we make this community-friendly within our approved processes?
```

Comment (mheadd) [reply adr/ADR-0007-opensearch-replacement-for-elasticsearch.md:39]: Not sure if this is what you intended, or if I'm missing the meaning of your original sentence.

Comment (bengerman13) [reply adr/ADR-0007-opensearch-replacement-for-elasticsearch.md:39]: funny is what I meant there, but it probably needs more context - most of which is here: https://github.com/cloud-gov/product/pull/1672/files

**Pull request closed without merging**